packages/bluebird/src/semantic-ui-react/_dropdown.scss

```diff
@@ -0,0 +1,1513 @@
+//   display: none;
+// }
+
+// /*--------------
+//      Sub Elements
+//   ---------------*/
+
+// /* Icons / Flags / Labels / Image */
+// .ui.dropdown > .text > .icon,
+// .ui.dropdown > .text > .label,
+// .ui.dropdown > .text > .flag,
+// .ui.dropdown > .text > img,
+// .ui.dropdown > .text > .image {
+//   margin-top: 0em;
+// }
+// .ui.dropdown .menu > .item > .icon,
+// .ui.dropdown .menu > .item > .label,
+// .ui.dropdown .menu > .item > .flag,
+// .ui.dropdown .menu > .item > .image,
+// .ui.dropdown .menu > .item > img {
+//   margin-top: 0em;
+// }
+.ui.dropdown > .text > .icon,
+.ui.dropdown > .text > .label,
+.ui.dropdown > .text > .flag,
+.ui.dropdown > .text > img,
+.ui.dropdown > .text > .image,
+.ui.dropdown .menu > .item > .icon,
+.ui.dropdown .menu > .item > .label,
+.ui.dropdown .menu > .item > .flag,
+.ui.dropdown .menu > .item > .image,
+.ui.dropdown .menu > .item > img {
+  //   margin-left: 0em;
+  //   float: none;
+  margin-right: $padding-small-horizontal;
+}
+
+// /*--------------
+//        Image
+//   ---------------*/
+
+.ui.dropdown > .text > img,
+.ui.dropdown > .text > .image,
+.ui.dropdown .menu > .item > .image,
+.ui.dropdown .menu > .item > img {
+  //   display: inline-block;
+  //   vertical-align: top;
+  //   width: auto;
+  margin-top: $padding-small-vertical;
+  margin-bottom: $padding-small-vertical;
+  //   max-height: 2em;
+}
+
+// /*******************************
+//               Coupling
+//   *******************************/
+
+// /*--------------
+//         Menu
+//   ---------------*/
+
+// /* Remove Menu Item Divider */
+// .ui.dropdown .ui.menu > .item:before,
+// .ui.menu .ui.dropdown .menu > .item:before {
+//   display: none;
+// }
+
+// /* Prevent Menu Item Border */
+// .ui.menu .ui.dropdown .menu .active.item {
+//   border-left: none;
+// }
+
+// /* Automatically float dropdown menu right on last menu item */
+// .ui.menu .right.menu .dropdown:last-child .menu,
+// .ui.menu .right.dropdown.item .menu,
+// .ui.buttons > .ui.dropdown:last-child .menu {
+//   left: auto;
+//   right: 0em;
+// }
+
+// /*--------------
+//         Label
+//   ---------------*/
+
+// /* Dropdown Menu */
+// .ui.label.dropdown .menu {
+//   min-width: 100%;
+// }
+
+// /*--------------
+//        Button
+//   ---------------*/
+
+// /* No Margin On Icon Button */
+// .ui.dropdown.icon.button > .dropdown.icon {
+//   margin: 0em;
+// }
+// .ui.button.dropdown .menu {
+//   min-width: 100%;
+// }
+
+// /*******************************
+//                 Types
+//   *******************************/
+
+// /*--------------
+//       Selection
+//   ---------------*/
+
+// /* Displays like a select box */
+.ui.selection.dropdown {
+  //   cursor: pointer;
+  //   word-wrap: break-word;
+  line-height: $line-height-small;
+  //   white-space: normal;
+  //   outline: 0;
+  //   -webkit-transform: rotateZ(0deg);
+  //   transform: rotateZ(0deg);
+  //   min-width: 14em;
+  min-height: 40px; // need variable
```

Comment:
Is this possibly a variable from forms/inputs? If not, define as custom semantic-ui variable at top of file.

Reply:
There is a variable, fixing this.